lib/Cake/Test/Case/Cache/CacheTest.php

```diff
@@ -519,4 +519,23 @@ public function testRemember() {
 	public function cacher() {
 		return 'This is some data ' . $this->_count;
 	}
+
+/**
+ * Test that failed add write return false.
```

Comment:
The title is somewhat misleading, as your test code not __only__ tests the failure case.